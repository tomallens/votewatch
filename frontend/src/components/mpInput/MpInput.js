import React from "react";
import { SafeAreaView, StyleSheet, TextInput, Text } from "react-native";

export const MpInput = () => {
  const [mpName, onChangeMpName] = React.useState("");

  return (
    <SafeAreaView>
      <TextInput
        style={styles.input}
        onChangeText={onChangeMpName}
        value={mpName}
        placeholder="Enter MP name"
      />
    </SafeAreaView>
  );
};

const styles = StyleSheet.create({
  input: {
    height: 40,
    margin: 12,
    borderWidth: 1,
    padding: 10,
  },
});
