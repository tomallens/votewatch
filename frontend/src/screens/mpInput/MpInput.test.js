import React from "react";
import TestRenderer from "react-test-renderer";
import { TextInput } from "react-native";

import { MpInput } from "./MpInput";

describe("MpInput", () => {
  test("renders correctly", () => {
    const tree = TestRenderer.create(<MpInput />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
