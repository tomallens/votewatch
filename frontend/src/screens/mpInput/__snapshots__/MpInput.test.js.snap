// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MpInput renders correctly 1`] = `
<View>
  <TextInput
    onChangeText={[Function]}
    placeholder="Enter MP name"
    style={
      {
        "borderWidth": 1,
        "height": 40,
        "margin": 12,
        "padding": 10,
      }
    }
    value=""
  />
</View>
`;
